From e6180e343a2f450160ade3aa9d7615bd8669ac39 Mon Sep 17 00:00:00 2001
From: Victor Queiroz <victorcqueirozg@gmail.com>
Date: Sat, 22 Mar 2025 13:13:52 -0300
Subject: [PATCH] Add support for `DESTDIR` environment variable

---
 lucidglyph.sh | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/lucidglyph.sh b/lucidglyph.sh
index 729819a..fa0d84c 100755
--- a/lucidglyph.sh
+++ b/lucidglyph.sh
@@ -27,18 +27,18 @@ SHOW_HEADER=${SHOW_HEADER:=true}
 
 # environment
 ENVIRONMENT_SCRIPT="$SRC_DIR/environment/lucidglyph.sh"
-DEST_ENVIRONMENT="/etc/environment"
+DEST_ENVIRONMENT="$DESTDIR/etc/environment"
 
 # fontconfig
 FONTCONFIG_DIR="$SRC_DIR/fontconfig"
-DEST_FONTCONFIG_DIR="/etc/fonts/conf.d"
+DEST_FONTCONFIG_DIR="$DESTDIR/etc/fonts/conf.d"
 #                    ("<NAME>" "<PRIORITY>")
 FONTCONFIG_GRAYSCALE=("lucidglyph-grayscale.conf" 11)
 FONTCONFIG_DROID_SANS=("lucidglyph-droid-sans.conf" 70)
 
 # Metadata location
-DEST_SHARED_DIR="/usr/share/lucidglyph"
-DEST_SHARED_DIR_OLD="/usr/share/freetype-envision"
+DEST_SHARED_DIR="$DESTDIR/usr/share/lucidglyph"
+DEST_SHARED_DIR_OLD="$DESTDIR/usr/share/freetype-envision"
 DEST_INFO_FILE="info"
 DEST_UNINSTALL_FILE="uninstaller.sh"
 
-- 
2.49.0

