From 5762ffeb534159a4c2ba34fdb2b9c3569d3df06c Mon Sep 17 00:00:00 2001
From: Victor Queiroz <victorcqueirozg@gmail.com>
Date: Sat, 22 Mar 2025 13:34:13 -0300
Subject: [PATCH] Add support for changing `DEST_ENVIRONMENT` environment

---
 lucidglyph.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lucidglyph.sh b/lucidglyph.sh
index fa0d84c..0618143 100755
--- a/lucidglyph.sh
+++ b/lucidglyph.sh
@@ -27,7 +27,9 @@ SHOW_HEADER=${SHOW_HEADER:=true}
 
 # environment
 ENVIRONMENT_SCRIPT="$SRC_DIR/environment/lucidglyph.sh"
-DEST_ENVIRONMENT="$DESTDIR/etc/environment"
+
+# If `DEST_ENVIRONMENT` is already defined, do not redefine it
+DEST_ENVIRONMENT="${DEST_ENVIRONMENT:-$DESTDIR/etc/environment}"
 
 # fontconfig
 FONTCONFIG_DIR="$SRC_DIR/fontconfig"
-- 
2.49.0

